Packs/RadwareCloudDDoSProtectionServices/Integrations/RadwareCloudDDoSProtectionServices/RadwareCloudDDoSProtectionServices_test.py

```diff
@@ -0,0 +1,187 @@
+import pytest
+from RadwareCloudDDoSProtectionServices import Client, fetch_events, fetch_alerts
+
+@pytest.fixture()
+def client() -> Client:
+    """
+    Create a Client instance for testing.
+    """
+    return Client(
+        base_url="test/api",
+        account_id="test_account_id",
+        api_key="test_api_key",
+        verify=False,
+        proxy=False,
+    )
+
+
+def test_fetch_events_no_last_run(mocker, client):
+    """
+    Given:
+     - no last_run (first time fetching).
+
+    When:
+     - running the fetch_events function.
+
+    Then:
+     - make sure events are fetched and processed correctly.
+     - make sure last_run is updated correctly.
```

Comment:
Let's refine the expectations: you expect for 2 events, that _time was added to the events and so on...
Let's go over the rest of the test to fix this